.travis.yml

```diff
@@ -81,4 +89,5 @@ script:
         (cd tests && make TESTARGS=-silent check -k
         TESTRUNNER=`which valgrind 2>/dev/null`)
   - make -s clean
-  - ./scripts/update-docs.sh
+  - [ $BUILD_DOCS ] && ./scripts/update-docs.sh
```

Comment:
We'll see what Travis says, but if `$BUILD_DOCS` is false, this entire line is false and thus could be considered a CI failure.

Reply:
Oh, yeah. Damn.  I put the || exit 0 there before.  I guess this maybe has to be if $BUILD_DOCS; then ... ; fi

Reply:
Or invert:
```yaml
 - [ $SKIP_DOCS ] || ./scripts/update-docs.sh
```